data/hai/hai missions.txt

```diff
@@ -1813,169 +1851,170 @@ mission "Nanachi Meeting"
 		conversation
-			`You enter the Remnant shipyard here and begin speaking with the shipwright about Remnant ships. One of the aspects of Remnant ships that the shipwright mentions to you is that they are able to repair their own hull. You should return to Hai-home and inform the Hai about this technology.`
-				decline
-
-mission "Strider: Coalition Hull Repair"
-	landing
-	source
-		government "Coalition" "Heliarch"
-		attributes outfitter
-	to offer
-		has "Strider: Find Hull Repair: active"
-		has "Coalition: First Contact: done"
-	on offer
+			`You enter the spaceport and find a Hai delegation ready to travel to the Remnant to negotiate for the hull repair technology. The leader of the delegation introduces herself as Ambassador Yashili. "I wasn't sure how far we will be travelling, so I packed assuming we will be travelling a great distance across the galaxy."`
+			choice
+				`	"Wait a minute! This group doesn't react well to strangers. How about I setup a meeting first?"`
+				`	"This is a bad idea. Sorry, but I am backing out."`
+					decline
+			`	The Hai looking at each other and chatter quietly among themselves, then Ambassador Yashili speaks up. "You know this group, and how to make the best impression on them." She pauses to whisper to another Hai, who runs off. The ambassador turns back to you. "I will provide you with a letter with a brief introduction to us, an explanation of why we would like to meet them, and an invitation to establish diplomatic relations."`
+			`	Moments later, the second Hai runs back into the spaceport carrying a sealed envelope and hands it to the Ambassador. She turns to you. "Here it is. Please take this to these reclusive humans for us."`
+				accept
+	on stopover
 		conversation
-			`While looking at the outfits that the Coalition have for sale, the hull repair module catches your eye. It uses a small robot to repair the hull of a ship while in flight. You should return to Hai-home and inform the Hai about this technology.`
-				decline
+			`Looking around the spaceport, you are not sure who to talk to about such a request. Eventually you go to the starport information desk to ask them. After a few minutes they tell you that someone will meet with you, and invite you to wait in the cafeteria.`
+			`	After an hour of watching Remnant eating and gesturing to each other, an androgynous Remnant approaches your table. You try to decide what age they are, but while they do not seem young, neither do they seem particularly old.`
+			branch signlanguage
+				has "Remnant: Learn Sign Follow Up: offered"
+			`	"Greetings, Captain <first>." The stranger chants. "My name is Rayn, and my colleague said you had a request from another faction?"`
+				goto stridercontinue
+			label signlanguage
+			`	"Greetings, Captain <first>." The stranger signs. "My name is Rayn, and my colleague said you had a request from another faction?"`
+			label stridercontinue
+			choice
+				`	"The Hai are hoping to learn from someone."`
```

Comment:
Is it established to the player before now that the Remnant 1) know the Hai, and 2) know enough about them that it's not surprising the Remnant immediately know about their conflict? I don't have the strongest grasp on the mission structure in relation to the other Remnant missions, so just thought it would be worth checking.

Reply:
It may or may not be established with the player that the Remnant know about the Hai, however the two resulting situations that I see are:
a) That the player is surprised that the Remnant know about the Hai, and
b) That the player has had a few hints that the Remnant are very well travelled and probably know about the Hai.

I don't see a problem with either situation. Either they didn't know the Remnant knew about them, and this is their chance to find out; or they did, and this is just normal. Either way works.

Reply:
Fair enough. I'd agree with your assessment.